spacy/errors.py

```diff
@@ -539,6 +539,7 @@ class Errors(object):
     E199 = ("Unable to merge 0-length span at doc[{start}:{end}].")
 
     # TODO: fix numbering after merging develop into master
+    E970 = ("Can not execute command '{str_command}'. Do you have '{tool}' installed?")
```

Comment:
This is a nice idea! 💯

Maybe we should rephrase this slightly to account for cases like `./non_existing_script.sh`? Although, I really like the "Do you have X installed?" message, which will likely be accurate in th majority of cases.